hadoop-tools/hadoop-azure/dev-support/testrun-scripts/testsupport.sh

```diff
@@ -63,6 +64,35 @@ fnsBlobConfigFileCheck() {
   fi
 }
 
+if ! command -v az &> /dev/null
+then
+  echo "Azure CLI (az) could not be found. Installing Azure CLI..."
+sudo apt update
+sudo apt install -y azure-cli
+echo "Azure CLI installed successfully."
+fi
+
+uploadToAzure() {
+  azureConfigFilePath="${accountSettingsDir}runresult${accountConfigFileSuffix}"
+  testResultsAccountName=$(xmlstarlet sel -t -v '//property[name = "fs.azure.test.results.account.name"]/value' -n $azureConfigFilePath)
+  testResultsAccountKey=$(xmlstarlet sel -t -v '//property[name = "fs.azure.test.results.account.key"]/value' -n $azureConfigFilePath)
+  branchName="${branchName,,}"
```

Comment:
Are the two commas needed here ?

Reply:
They are used to convert into lowercase letters in Bash